Makefile

```diff
@@ -82,8 +85,8 @@ images:  ## Build tmt images for podman/docker
 ##
 ## Development
 ##
-develop:  ## Install development requirements
-	sudo dnf --setopt=install_weak_deps=False install hatch gcc make git rpm-build python3-nitrate {python3,libvirt,krb5,libpq}-devel jq podman
+develop: minideps  ## Install development requirements
```

Comment:
I might be late to the party, but: `minideps`, `deps`, might be confusing (it certaiňy us to me…). If `deps` is for installing buikd deps, would `build-deps` be a bad name? Plus “library” targets often start with dot or underscore, how sbout `_minideps`, or `_deps` (with `deps` replaced by `build-deps`, there would be no conflict).

Reply:
The `minideps` target is just a helper target for the other two. I intentionally haven't added it to the `make help`. So, yes, agreed, we can prefix it with an underscore.

Regarding `deps` versus `build-deps`, the first one is shorter, the second one clearer, don't know :) @lukaszachy, @martinhoyer, any quick votes/preferences?

Reply:
No vote from me as I don't know much about Makefiles to have an opinion.

Reply:
Thanks, should be covered by 316fab3d.